src/popup/style.css

```diff
@@ -1,18 +1,27 @@
 body {
+    background: #eee;
```

Comment:
```suggestion
    background: #fafbfc;
```
This is the official color from Primer style (used on Github if you look at the header bar on this page)

Reply:
I've changed the color